ceph.spec.in

```diff
@@ -1636,6 +1636,8 @@ fi
 %{_bindir}/ceph-osdomap-tool
 %{_bindir}/ceph-kvstore-tool
 %{_bindir}/ceph-debugpack
+%{_bindir}/ceph_omap_create
```

Comment:
nit, i'd suggest use dash instead of underscore to be easier to type and to be more consistent with other tools' names.